front-end/src/components/PostListItem.tsx

```diff
@@ -13,6 +13,10 @@ interface Props {
   removeFromFeed: Function;
 }
 
+/**
+ * Post list item component to show a preview for an individual post on a stream
```

Comment:
I think the PostContentEl is the preview. The comment could be like: "PostListItem component is an individual post list item on a stream" or something like this.